app/experimenter/nimbus-ui/src/components/TableVisualizationRow/index.tsx

```diff
@@ -178,12 +178,19 @@ const TableVisualizationRow: React.FC<{
     className = tooltipText = "";
     const percent = branch_data[METRIC.USER_COUNT]["percent"];
     const userCountMetric =
-      branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE]["point"];
+      branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE].length > 0
+        ? branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE][0]["point"]
```

Comment:
uber nit: Some of this is getting tricky to read. Not sure on the collective feelings on using a temp variable to reduce code duplication.

```js
const bData = branch_data[METRIC.USER_COUNT][BRANCH_COMPARISON.ABSOLUTE];
const userCountMetric = bData.length ? bData[0].point : null;
```